### PR TITLE
Update with details of new SIGs

### PR DESCRIPTION
**Pull request by ewilderj**

### CODE REVIEW

File: site/en/community/contributing.md [hunk 1]
```diff
@@ -39,8 +39,14 @@ Special Interest Groups (SIGs). SIGs do their work in public: if you want to
 join and contribute, review the work of the group, and get in touch with the
 relevant SIG leader.  Membership policies vary on a per-SIG basis.
 
+* **SIG Addons** maintains a repository of contributions that conform to well-established API patterns, but implement new functionality not available in core TensorFlow. [Mailing list](https://groups.google.com/a/tensorflow.org/d/forum/addons).
+
 * **SIG Build** focuses on issues surrounding building, packaging, and
   distribution of TensorFlow. [Mailing list](https://groups.google.com/a/tensorflow.org/d/forum/build).
+  
+* **SIG IO** works on support for file systems and formats other than those supported in core TensorFlow, as subclasses of tf.data.Dataset and TensorFlow filesystems, and supporting code. [Mailing list](https://groups.google.com/a/tensorflow.org/d/forum/io)
```
Comment (MarkDaoust) [inline site/en/community/contributing.md]: >supported in core TensorFlow, as subclasses

such as?

Comment (ewilderj) [reply site/en/community/contributing.md]: such as Apache Ignite File System. @martinwicke any others we could mention as examples for SIG IO?

Comment (MarkDaoust) [reply site/en/community/contributing.md]: Sorry Edd, what I meant to say was:

"The wording here seemes a little funny. Should that line read '...core TensorFlow, **such as** subclasses...'"

Not: "Can you give examples of this"

Comment (ewilderj) [reply site/en/community/contributing.md]: _Ohhh_ hehe. Well now I went and added some examples in. And I cleaned the language up too! See what you think

Comment (yongtang) [reply site/en/community/contributing.md]: @ewilderj Apache Kafka and Apache Parquet (https://github.com/tensorflow/tensorflow/pull/19461) are some other examples.

**Comment by ewilderj** [comment]
Thanks @MarkDaoust - what's the workflow here now, do I merge the PR, will somebody else?

**Comment by MarkDaoust** [comment]
Oh, either way's fine.

**Comment by ewilderj** [comment]
Thanks!

**Pull request merged**